js/background.js

```diff
@@ -1183,15 +1227,17 @@ get_l10n_data = (SAFARI ? chrome.i18n._getL10nData : undefined);
 
 // BROWSER ACTION AND CONTEXT MENU UPDATES
 (function() {
-    if (SAFARI)
+    if (SAFARI) {
         return;
+    }
```

Comment:
Readability: could the entire function become wrapped in an `if (!SAFARI)` block?